src/containers/nav-bar.html

```diff
@@ -37,24 +49,32 @@
 
 <aside class="mdc-drawer mdc-drawer--modal">
```

Comment:
Can we make the nav drawer open on the right side of the screen? It's a little weird that you click the button on the right side and it opens a panel on the left.

Reply:
Do you know of a way to open the panel on the right? I looked through their documentation and didn't see anything that allowed for that. I could manually override the styles and animations but that might be the only way.

Reply:
I just assumed it would be a built in feature of the drawer. I guess not. This guy seems to accomplish it with this css: https://github.com/material-components/material-components-web/issues/1324#issuecomment-557786833

if that doesn't work, don't spend too much time on it.